docs/architecture/adr-056-proving-amnesia-attack.md

```diff
@@ -0,0 +1,85 @@
+# ADR 56: Proving Amnesia Attack
+
+## Changelog
+
+- 02.04.20: Initial Draft
+
+## Context
+
+Whilst most created evidence of malicious behaviour is self evident such that any individual can verify them independently there are types of evidence that require collaboration from the network of validators, known as global evidence, in order to accumulate enough information to create evidence that is individually verifiable and can therefore be processed through consensus. Fork Accountability as a whole constitutes of detection, proving and punishing. This ADR addresses how to prove infringement from global evidence.
+
+So far, only [flip flopping](https://github.com/tendermint/spec/blob/master/spec/consensus/light-client/accountability.md#flip-flopping) attacks such as amnesia (which will be what the reactor initially focuses on) and back to the past (only causes light forks) have been identified as forms of global evidence that require more information from other validators.
```

Comment:
What is a light fork?

Reply:
artificially constructed fork, created to fool a particular light client(s)